mycore.org/content/de/documentation/getting_started/gs_jpa.html

```diff
@@ -2,194 +2,404 @@
+
+        <mapping-file>META-INF/mycore-base-mappings.xml</mapping-file>
+        <mapping-file>META-INF/mycore-ifs-mappings.xml</mapping-file>
+        <mapping-file>META-INF/mycore-iview2-mappings.xml</mapping-file>
+        <mapping-file>META-INF/mycore-pi-mappings.xml</mapping-file>
+        <mapping-file>META-INF/mycore-user2-mappings.xml</mapping-file>
+        <mapping-file>META-INF/mycore-jobqueue-mappings.xml</mapping-file>
+
+        <properties>
+
+            <!--  PostgreSQL -->
+            <property name="jakarta.persistence.jdbc.driver" value="org.postgresql.Driver"/>
+            <property name="jakarta.persistence.jdbc.url"
+                      value="jdbc:postgresql://127.0.0.1/MY_DATABASE?currentSchema=MY_SCHEMA"/>
+
+            <property name="jakarta.persistence.jdbc.user" value="DATABASE_USER"/>
+            <property name="jakarta.persistence.password" value="DATABASE_PASSWORD"/>
+
+            <property name="hibernate.default_schema" value=""/>
+            <property name="hibernate.hbm2ddl.auto" value="update"/>
+
+            <!-- second-level cache  -->
+            <property name="hibernate.cache.region.factory_class"
+                      value="org.hibernate.cache.internal.NoCachingRegionFactory"/>
+            <property name="hibernate.cache.use_second_level_cache" value="false"/>
+            <property name="hibernate.cache.use_query_cache" value="false"/>
+
+            <!-- Set batch_size to "0" in case of SQL batch Errors -->
+            <property name="hibernate.jdbc.batch_size" value="2000"/>
+            <property name="hibernate.default_batch_fetch_size" value="100"/>
+
+            <!-- Echo all executed SQL to stdout -->
+            <property name="hibernate.show.sql" value="true"/>
+            <property name="hibernate.format.sql" value="true"/>
+
+            <!-- Homepage: https://github.com/brettwooldridge/HikariCP -->
+            <property name="hibernate.connection.provider_class" value="org.hibernate.hikaricp.internal.HikariCPConnectionProvider" />
+            <property name="hibernate.hikari.maximumPoolSize" value="30" />
+            <property name="hibernate.hikari.leakDetectionThreshold" value="9000" />
+            <property name="hibernate.hikari.registerMbeans" value="true" />
+
+        </properties>
+    </persistence-unit>
 </persistence>
 {{< /highlight >}}
 {{< /mcr-figure >}}
 
-      <h3>Hinweis bei Performance-Problemen</h3>
-  
-      <p>Sollt die Anwendung wiederholt abstürzen, da es mit der Anzahl der freien Datenbankverbindungen 
-      Ausnahmen gibt, könnte folgende zusätzliche Konfiguration helfen. Tote Sessions werden dann automatisch geschlossen.</p>
-  
-{{< mcr-figure label="Konfigurationsdatei" caption="<code>resources/META-INF/persistence.xml</code>">}}
+<p>
+    Außerdem muss für die Verwendung von HikariCP der Treiber in der pom.xml der Anwendung hinzugefügt werden.
+</p>
+
+{{< mcr-figure label="Konfigurationsdatei" caption="<code>pom.xml</code>">}}
```

Comment:
"Maven Build Datei" oder Builddatei" als label, ggf. nochmal andere, bestehende abels für pom.xml in unserer Doku suchen.